app/views/contractuals/_form.html.erb

```diff
@@ -27,6 +27,6 @@
   </div>
 
   <div class="actions">
-    <%= form.submit t('contractual.button_send') %>
+    <%= form.submit t('contractual.button_send'), { :class => 'btn'} %>
```

Comment:
This also needs to be changed, overall LGTM 🤓 👍

Reply:
haaa se me paso, deja lo cambio